fastlane/docs/Actions.md

```diff
@@ -1280,6 +1280,37 @@ You can also only receive the version number without modifying it
 version = get_version_number(xcodeproj: "Project.xcodeproj")
 ```
 
+### increment_version_number_in_plist
```

Comment:
How is this different from `increment_version_number`? That we don't use the Xcode feature to increment it?